zerver/migrations/0455_set_default_value_for_create_multiuse_invite_group.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 4.2.1 on 2023-06-03 10:53
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def set_default_value_for_create_multiuse_invite_group(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    UserGroup = apps.get_model("zerver", "UserGroup")
+
+    Realm.POLICY_NOBODY = 6
+    Realm.POLICY_ADMINS_ONLY = 2
+
+    UserGroup.NOBODY_GROUP_NAME = "@role:nobody"
+    UserGroup.ADMINISTRATORS_GROUP_NAME = "@role:administrators"
+    default_group_name = "@role:moderators"
+
+    for realm in Realm.objects.all():
+        if realm.create_multiuse_invite_group is None:
+            if realm.invite_to_realm_policy is Realm.POLICY_NOBODY:
```

Comment:
Would be better to use `==` for comparing values here. We generally use `is` for checking if  value is `None` or not.